aiohttp/cookiejar.py

```diff
@@ -81,14 +99,8 @@ def __init__(
                 for url in treat_as_secure_origin
             ]
         self._treat_as_secure_origin = treat_as_secure_origin
-        self._next_expiration = next_whole_second()
-        self._expirations: Dict[Tuple[str, str, str], datetime.datetime] = {}
-        # #4515: datetime.max may not be representable on 32-bit platforms
-        self._max_time = self.MAX_TIME
-        try:
-            self._max_time.timestamp()
-        except OverflowError:
-            self._max_time = self.MAX_32BIT_TIME
+        self._next_expiration: Union[int, float] = ceil(time.time())
```

Comment:
```suggestion
        self._next_expiration: float = ceil(time.time())
```
int is compatible with float